hdc/algo/ops/stats.py

```diff
@@ -231,7 +235,8 @@ def gammastd_yxt(
                 continue
             s = gammastd(xt, nodata, cal_start, cal_stop)
             if (s != nodata).sum() > 0:
-                s = s * 1000
+                valid_ix = s != nodata
+                s[valid_ix] = s[valid_ix] * 1000
```

Comment:
I wonder if it'd be better to loop here instead of using indexing - did you try / compare?